### PR TITLE
Update `Comment.text` to `Comment.body`

### PR DESCRIPTION
**Pull request by timolegros**
## Link to Issue
Closes: #9673 

## Description of Changes
- Renames `Comment.text` to `Comment.body` and `CommentVersionHistories.text` to `CommenVersionHistories.body`

## Test Plan
- Search for comments
- View global/for-you dashboards
- View Overview and All thread pages
- View specific threads with comments
- View comment histories

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

**Review by Rotorsoft — APPROVED** [review]

File: packages/commonwealth/client/scripts/models/Thread.ts [hunk 1]
```diff
@@ -462,7 +462,7 @@ export class Thread implements IUniqueId {
           parent_id: null,
           reactions: [],
           CommentVersionHistories: [],
-          reaction_weights_sum: '0',
```
Comment (mzparacha) [inline packages/commonwealth/client/scripts/models/Thread.ts:465]: i think this was intentional in https://github.com/hicommonwealth/commonwealth/commit/7e56b0c11d4cc41e3b746d19940985ea250b26c1#diff-65f6a84029354b4ea2d8a68ac1b9453ee76725ba26af0e19af0e1f582b9083d2

cc: @rbennettcw to confirm.

**Review by mzparacha — COMMENTED** [review]
Few more instances where `.text` is still referenced. Some of these don't error coz of the stick null check disable comment
- https://github.com/hicommonwealth/commonwealth/blob/afeefe9fa7f7b6b89701c2da568d109d3bc37870/packages/commonwealth/client/scripts/views/components/Profile/ProfileActivityRow.tsx#L143
- https://github.com/hicommonwealth/commonwealth/blob/afeefe9fa7f7b6b89701c2da568d109d3bc37870/packages/commonwealth/client/scripts/views/components/UpvoteDrawer/ViewCommentUpvotesDrawer.tsx#L37
- https://github.com/hicommonwealth/commonwealth/blob/80a7b4038d86e3e482eb49570dfc1c8f0421aecb/packages/commonwealth/client/scripts/views/pages/discussions/CommentCard/CommentCard.tsx#L106-L108
- https://github.com/hicommonwealth/commonwealth/blob/8a494fdb97f099569098b3c829012b465efffdd6/packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx#L329
- https://github.com/hicommonwealth/commonwealth/blob/8a494fdb97f099569098b3c829012b465efffdd6/packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx#L353
- https://github.com/hicommonwealth/commonwealth/blob/8a494fdb97f099569098b3c829012b465efffdd6/packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx#L329
- https://github.com/hicommonwealth/commonwealth/blob/2670a16b8d145cb78a04dfc27da7a5a060ae46f1/packages/commonwealth/client/scripts/models/Thread.ts#L446
- https://github.com/hicommonwealth/commonwealth/blob/80a7b4038d86e3e482eb49570dfc1c8f0421aecb/packages/commonwealth/client/scripts/models/Comment.ts#L97-L99

**Comment by timolegros** [comment]
> Few more instances where `.text` is still referenced. Some of these don't error coz of the stick null check disable comment
> 
>     * https://github.com/hicommonwealth/commonwealth/blob/afeefe9fa7f7b6b89701c2da568d109d3bc37870/packages/commonwealth/client/scripts/views/components/Profile/ProfileActivityRow.tsx#L143
> 
>     * https://github.com/hicommonwealth/commonwealth/blob/afeefe9fa7f7b6b89701c2da568d109d3bc37870/packages/commonwealth/client/scripts/views/components/UpvoteDrawer/ViewCommentUpvotesDrawer.tsx#L37
> 
>     * https://github.com/hicommonwealth/commonwealth/blob/80a7b4038d86e3e482eb49570dfc1c8f0421aecb/packages/commonwealth/client/scripts/views/pages/discussions/CommentCard/CommentCard.tsx#L106-L108
> 
>     * https://github.com/hicommonwealth/commonwealth/blob/8a494fdb97f099569098b3c829012b465efffdd6/packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx#L329
> 
>     * https://github.com/hicommonwealth/commonwealth/blob/8a494fdb97f099569098b3c829012b465efffdd6/packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx#L353
> 
>     * https://github.com/hicommonwealth/commonwealth/blob/8a494fdb97f099569098b3c829012b465efffdd6/packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx#L329
> 
>     * https://github.com/hicommonwealth/commonwealth/blob/2670a16b8d145cb78a04dfc27da7a5a060ae46f1/packages/commonwealth/client/scripts/models/Thread.ts#L446
> 
>     * https://github.com/hicommonwealth/commonwealth/blob/80a7b4038d86e3e482eb49570dfc1c8f0421aecb/packages/commonwealth/client/scripts/models/Comment.ts#L97-L99

This is intentional. I'm trying to keep the surface area on the client as small as possible. I only updated the client where it was necessary. I used the `Comment` client-side model to use `body` instead of `text` if it exists. The rest of the `text` fixes should be completed when the client-side `Comment` model is removed.

File: packages/commonwealth/client/scripts/models/Thread.ts (hunk 1 above)
Comment (timolegros) [reply packages/commonwealth/client/scripts/models/Thread.ts:465]: oops yea will revert that

**Review by mzparacha — APPROVED** [review]

**Pull request merged**